hoomd/tune/custom_tuner.py

```diff
@@ -26,9 +26,26 @@ def tuner(self, tuner):
 
 
 class CustomTuner(CustomOperation, _TunerProperty, Tuner):
-    """Tuner wrapper for `hoomd.custom.Action` objects.
+    """User-defined writer.
 
-    For usage see `hoomd.custom.CustomOperation`.
+    Args:
+        action (hoomd.custom.Action): The action to call.
+        trigger (hoomd.trigger.Trigger): Select the timesteps to call the
+          action.
+
+    `CustomTuner` is a `hoomd.operation.Tuner` that wraps a user-defined
+    `hoomd.custom.Action` object  so it can be added to the `hoomd.Simulation`'s
+    `hoomd.Operations` and called during the run.
```

Comment:
```suggestion
    `hoomd.custom.Action` object  so the action can be added to a
    `hoomd.Operations` instance for use with `hoomd.Simulation` objects.
```
I think this suggestion is a bit clearer to the intent of the statement (from the user perspective at least if not the developmental reasons for the wrapping). If you agree change this for all of the subclasses. If not just resolve the issue and leave as is.